sysl2/sysl/Parser_test.go

```diff
@@ -111,171 +154,181 @@ func parseAndPrint(t *testing.T, filename, root string) error {
 	return nil
 }
 
-func testParse(t *testing.T, filename, root string) {
-	equal, err := parseAndCompare(filename, root)
+func testParseAgainstPython(t *testing.T, filename, root string) {
+	equal, err := parseAndCompareWithPython(filename, root)
+	assert.NoError(t, err)
+	assert.True(t, equal, "Mismatch")
+}
+
+func testParseAgainstGolden(t *testing.T, filename, root string) {
```

Comment:
Similar here, except root is never non-empty.

Reply:
As above.